vulcanai2/models/layers.py

```diff
@@ -62,9 +63,14 @@ def _init_bias(self):
         if self.bias_init is None, then pytorch default bias
         will be assigned to the kernel
         """
-        if self.bias_init:
+        if self.bias_init is not None:
```

Comment:
@joemehltretter I thought you were going to change it to passing the bias_init a function to use to initialize - the same way that the weight is initialized

Reply:
I must've forgot that, my bad. I'll do now.

Reply:
now that bias_init is a function, can you revert it back to `if self.bias_init`?